teeplot/teeplot.py

```diff
@@ -377,3 +382,76 @@ def teed(*args: list, **kwargs: dict):
         yield handle
     finally:
         saveit()
+
+
+def validate_teewrap_kwargs(teeplot_kwargs):
+    params = {k for k in inspect.signature(tee).parameters if k.startswith("teeplot")}
```

Comment:
I would just assert that all the kwargs start with teeplot_, if users pass, invalid teeplot_ kwargs they’ll get an error on the call to tee already, which isn’t too hard to decipher.

Reply:
i.e. we don’t need to inspect signature